sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample06_IdentityAndSharedAccessCredentials.md

```diff
@@ -2,6 +2,19 @@
 
 This sample demonstrates using credentials to authorize clients with the Event Hubs service.  In most scenarios, credentials provide a better approach to security than connection strings for production applications.  One of the key benefits is support for in-place rotation to update access policies without the need to stop applications.  Most credential sources also allow for more fine-grained access control, when compared with the shared key approach favored by connection strings.
 
+## Table of contents
+
+- [Prerequisites](#prerequisites)
+    - [Identity authorization](#identity-authorization)
+    - [Event Hubs Shared Access Signature authorization](#event-hubs-shared-access-signature-authorization)
+    - [Event Hubs Shared Access Key authorization](#event-hubs-shared-access-key-authorization)
+- [Client types](#client-types)
+- [Publishing events with identity-based authorization](#publishing-events-with-identity-based-authorization)
+- [Publishing events with Shared Access Signature authorization](#publishing-events-with-shared-access-signature-authorization)
+- [Publishing events with Shared Access Key authorization](#publishing-events-with-shared-access-key-authorization)
+- [Parsing a connection string for information](#parsing-a-connection-string-for-information)
+
+
```

Comment:
```suggestion
```

Reply:
Ahh! I'll drop that extra line. Good catch.